README.md

```diff
@@ -27,6 +27,28 @@
   </a>
 </p>
 
+## Roadmap
```

Comment:
I think we should also rephrase the opening paragraph to use a more active voice (i.e. "looking for the official dashboard? it still exists, but the code has moved"). It's not that there's no dashboard anymore, it's just not open source any more (for reasons described in the Discourse post). 

> As of version 6.0, the official Sensu web app has moved to the commercial product codebase. Users who want an officially supported web app for use with Sensu Go should try the commercial distribution available at https://sensu.io/downloads 
>
> Users who download and compile Sensu Go from source and want a web UI will need to download and run this web app as a separate component (see [INSTALL.md](INSTALL.md)). Please note that the web interface included in the commercial distribution no longer shares a common codebase with this project. For more details regarding these changes, please see [Building a better UI for Sensu].